virttest/utils_net.py

```diff
@@ -1540,6 +1540,38 @@ def set_guest_ip_addr(session, mac, ip_addr,
         raise IPAddrSetError(mac, ip_addr, err)
 
 
+def get_guest_default_gateway(session):
```

Comment:
I would suggest to refactor `get_host_default_gateway()` with session to support for guest,
https://github.com/avocado-framework/avocado-vt/blob/f3fabed182206e279ec6b88eabcff098ff755003/virttest/utils_net.py#L3745-L3753

Reply:
Thank you! That's good as they can share the same command to get the default gateway. But the name of the function will need to be changed to "get_default_gateway", then more files which call "get_host_default_gateway" will need to be updated in tp-libvirt. CI will not be so stable in that period.

Reply:
correct! better to change the name of the function, it is used only once in avocado-vt/virttest/utils_net.py and in tp-qemu/qemu/tests/mq_change_qnum.py. let's have the change.

Reply:
Updated, please help to check, Thank you!

Reply:
update the tp-libvirt:
https://github.com/autotest/tp-libvirt/pull/2380
update the tp-qemu:
https://github.com/autotest/tp-qemu/pull/1962